AlgebraInLean/Chapter01/Sheet02.lean

```diff
@@ -4,10 +4,11 @@ namespace AlgebraInLean
 
 /-
 Groups aren't the only important structures in algebra. By including or excluding certain properties
-from our definition, we create different structures.
+from our definition, we can create different structures.
```

Comment:
I believe that having "create" be the verb here works well enough. To change it to "can", I think we'd need to change the previous clause to "By choosing to include or exclude[]", which imo is overly verbose.

Reply:
Agreed